workflow-service-sdk/pom.xml

```diff
@@ -17,6 +17,8 @@
         <spring.javaformat.version>0.0.34</spring.javaformat.version>
         <openapi.generator.version>5.4.0</openapi.generator.version>
         <lombok.version>1.18.26</lombok.version>
+        <skip.sdk.generation>false</skip.sdk.generation>
+        <!--        <openapi.unchanged>false</openapi.unchanged>-->
```

Comment:
do we need this comment?

Reply:
No, it's unnecessary. Deleted. 
Thanks!